tests/units/cli/test_examples/kubernetes-atomicapp/answers.conf

```diff
@@ -0,0 +1,4 @@
+[kubernetes-atomicapp]
+[general]
+namespace = kubernetes
+provider = docker
```

Comment:
ahh I see.. This is actually to run kubernetes in containers using docker.. Nevermind.